.gitignore

```diff
@@ -1,5 +1,5 @@
 minivpn
-./obfs4vpn
+obfs4vpn
```

Comment:
Why did this change? Could we have the binary at arbitrary directories?

Reply:
it actually should be removed, the cmd/obfs4vpn endpoint was deleted.